business-central-tests/business-central-tests-gui/README.md

```diff
@@ -1,4 +1,6 @@
-# KIE Workbench GUI tests (selenium)
+<<<<<<< HEAD:business-central-tests/business-central-tests-gui/README.md
```

Comment:
^Seems like merge leftover in the readme.
Thanks for the merge and sorry for the complications :)
I'll test the consolidated business-central locally starting tomorrow and provide feedback asap.

Reply:
ooops, removed the leftover :)
np at all